ca.potatoe.cmdjewel.yml

```diff
@@ -0,0 +1,64 @@
+app-id: ca.potatoe.cmdjewel
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: cmdjewel
+finish-args:
+  - --socket=pulseaudio
+modules:
```

Comment:
if it's a desktop app how does it display things without x11 socket or wayland+fallback-x11 socket?

Reply:
> I wrote for the terminal.

In general terminal applications are not a good fit for flathub.

Reply:
the appdata has

> <component type="desktop-application">

Reply:
sorry about that! this was a bit confusing for me since cmdjewel's a terminal program in about the same sense of cmus, which has a flatpak. basically it's a program that displays on the terminal, but also is meant to be a graphical application? i copied over `type="desktop-application"` from cmus, but i'm not sure if i should be using `type="console-application"` instead.